contracts/contracts/ccip/router.tolk

```diff
@@ -21,12 +21,13 @@ fun Storage.store(self) {
     return contract.setData(self.toCell());
 }
 
-type Msg = CCIPSend | SetRamp | JettonTransferNotification;
+type Msg = CCIPSend | SetRamp | SetRamps | JettonTransferNotification;
```

Comment:
I'd remove/replace the old method, no need for both -- it just increases the bytecode size